presto/dataops/utils.py

```diff
@@ -27,6 +27,24 @@ def construct_single_presto_input(
     dynamic_world: Optional[torch.Tensor] = None,
     normalize: bool = True,
 ):
+    """
+    Inputs are paired into a tensor input <X> and a list <X>_bands, which describes <X>.
+
+    <X> should have shape (num_timesteps, len(<X>_bands)), with the following bands possible for
+    each input:
+
+    s1: ["VV", "VH"]
+    s2: ["B1", "B2", "B3", "B4", "B5", "B6", "B7", "B8", "B8A", "B9", "B10", "B11", "B12"]
```

Comment:
Likewise could be useful to mention here it's expected to be in scaled reflectance value (1E4).